mission/functions/core/ui/taskroster/fn_tr_characterInfo_ribbon_exit.sqf

```diff
@@ -18,19 +18,16 @@
 
 #include "..\..\..\..\config\ui\ui_def_base.inc"
 params["_ctrl_src"];
-_ctrl_preview = VN_TR_CHARINFO_MEDAL_RIBBON_CTRL;
+_ctrl_preview = VN_TR_PLAYERINFO_MEDAL_RIBBON_CTRL;
 //set img
 _icon = _ctrl_preview getVariable ["curIcon", ""];
 _ctrl_preview ctrlSetText _icon;
 //set description text
 _text = _ctrl_preview getVariable ["text", ""];
-VN_TR_CHARINFO_REWARD_TEXT_CTRL ctrlSetStructuredText parseText format["<t size='0.65' >%1</t>", _text];
+VN_TR_PLAYERINFO_REWARD_TEXT_CTRL ctrlSetStructuredText parseText format["<t size='0.65' >%1</t>", _text];
 //set tooltip (name)
 _tooltip = _ctrl_preview getVariable ["name", ""];
 _tooltip = if(isLocalized _tooltip)then{localize _tooltip}else{_tooltip};
 _ctrl_preview ctrlSetToolTip _tooltip;
 //re-adjust size
-_ctrl_size = _ctrl_preview getVariable ["size_cur",[UIH(17.65),UIH(1.10)]];
-_ctrl_preview ctrlSetPositionY _ctrl_size#0;
-_ctrl_preview ctrlSetPositionH _ctrl_size#1;
```

Comment:
as per `mission/functions/core/ui/taskroster/fn_tr_characterInfo_ribbon_enter.sqf` changing the location and size of the medal preview after interaction caused a bit of a headache considering everything else has a static position defined in a config